doc-content/enterprise-only/project-data/delete-branches-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='delete-branches-proc']
+
+= Deleting branches
+You can delete any branch, except for the *master* branch because {CENTRAL} does not allow you to delete the *master* branch to avoid corrupting your environment. You must be in any branch other than *master* for the following procedure to work.
```

Comment:
The first sentence here really does not sound right to me, style-wise.

Reply:
Yeah, that one sucks. Reworked.